media/stylus/components/home/column-callout.styl

```diff
@@ -0,0 +1,104 @@
+/*
+colorful callouts
+====================================================================== */
+$callout-icon-spacing = 25px;
+$callout-icon-width = 90px;
+
+.column-callout    {
```

Comment:
Just a code style nitpick. Maybe just one space between the selector and the brace?